scrapy/utils/boto.py

```diff
@@ -0,0 +1,20 @@
+"""Boto/botocore helpers"""
+
+import six
+
+from scrapy.exceptions import NotConfigured
+
+
+def is_botocore():
+    try:
+        import botocore
+        return True
+    except ImportError:
+        if six.PY2:
+            try:
+                import boto
```

Comment:
I can't confidently tell if this imports boto from site-packages or from scrapy.utils.boto. What about adding `absolute_imports` future import to this file?